classes/class3/hw/app.js

```diff
@@ -0,0 +1,27 @@
+var express = require('express');
+var path = require('path');
+var logger = require('morgan');
+var exphbs = require('express-handlebars');
+var mongoose = require('mongoose');
+var funs = require('./funs');
```

Comment:
Putting a bunch of unrelated functions in a file and requiring it kind of misses the point of using `require`.